### PR TITLE
vcreate: minor tweaks

### PR DESCRIPTION
**Pull request by StunxFS**
This PR makes minor tweaks to `vcreate.v`.

### CODE REVIEW

**Review by serkonda7 — APPROVED** [review]

File: cmd/v/help/init.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Usage:
+  v init
+
+Setup the file structure for an already existing V project.
```
Comment (serkonda7) [inline cmd/v/help/init.txt:4]: please explain in a little more detail what files exactly will be created

**Review by serkonda7 — CHANGES_REQUESTED** [review]

File: cmd/v/help/init.txt (hunk 1 above)
Comment (StunxFS) [reply cmd/v/help/init.txt:4]: Okey c:

**Comment by RiordanIX** [comment]
Also, why can't the name be capitalized?

**Comment by spytheman** [comment]
> Also, why can't the name be capitalized?

Because the name can be later used as a module name, and you can not have `import Capital`.

**Pull request merged**